server/src/concepts/user/repositories/queries.js

```diff
@@ -0,0 +1,13 @@
+import User from "../model/User";
+
+export async function getUser(userId) {
+  const user = await User.findById(userId);
```

Comment:
What if we do not fundusz a user with The given id? Shouldn't there be an if statement for that case?